tests/unit/test_matrix.py

```diff
@@ -256,3 +256,9 @@ def it_returns_correct_base_values(self, _cube_result_matrix_prop_, prop_name):
     @pytest.fixture
     def _cube_result_matrix_prop_(self, request):
         return property_mock(request, Assembler, "_cube_result_matrix")
+
+
+class Describe_CatXCatMatrix(object):
+    def it_provides_its_unweighted_counts_to_help(self):
```

Comment:
This one doesn't get "_to_help" because the `.unweighted_counts` property is interface, not a helper.

Reply:
sure. but this already got fixed